azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/amqp/ConnectionHandler.java

```diff
@@ -147,7 +147,7 @@ public void onConnectionLocalClose(Event event) {
 
         final ErrorCondition error = connection.getCondition();
         if (TRACE_LOGGER.isInfoEnabled()) {
-            TRACE_LOGGER.info("hostname[" + connection.getHostname() +
+            TRACE_LOGGER.info("Connection.onConnectionLocalClose: hostname[" + connection.getHostname() +
```

Comment:
Feel free to remove connection. prefix - this was the original name of this class and is no longer true.